<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>392</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>388</width>
    <height>392</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pintrest 이미지 크롤링 봇</string>
  </property>
  <property name="iconSize">
   <size>
    <width>48</width>
    <height>48</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>370</width>
        <height>51</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>370</width>
        <height>51</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QPushButton" name="executeButton">
         <property name="minimumSize">
          <size>
           <width>41</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>41</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>동작</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLabel" name="boardLabel">
         <property name="minimumSize">
          <size>
           <width>85</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="stopButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>정지</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="statusLabel">
         <property name="minimumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>상태</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="statusSignal">
         <property name="minimumSize">
          <size>
           <width>41</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>41</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">Color:red</string>
         </property>
         <property name="text">
          <string>●</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="directButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>71</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>71</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>즉시실행</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>저장하기</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="minimumSize">
       <size>
        <width>370</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>370</width>
        <height>45</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QLineEdit" name="password_line_edit">
         <property name="minimumSize">
          <size>
           <width>125</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>125</width>
           <height>25</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Pintrest 비밀번호</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLineEdit" name="member_id_line_edit">
         <property name="minimumSize">
          <size>
           <width>125</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>125</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Pintret 아이디</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="registerButton">
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>81</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>등록하기</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>동작시간</string>
      </property>
      <widget class="QCheckBox" name="time00Hour">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>43</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>00시</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="time01Hour">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>43</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>01시</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="time02Hour">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>43</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>02시</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="time03Hour">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>40</y>
         <width>43</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>03시</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="time04Hour">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>43</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>04시</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="time05Hour">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>60</y>
         <width>43</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>05시</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="time06Hour">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>43</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>06시</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="time07Hour">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>43</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>07시</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="time08Hour">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>43</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>08시</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="time09Hour">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>100</y>
         <width>43</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>09시</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="time10Hour">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>43</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>10시</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="time11Hour">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>120</y>
         <width>43</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>11시</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="time12Hour">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>43</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>12시</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="time13Hour">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>140</y>
         <width>43</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>13시</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="time14Hour">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>43</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>14시</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="time15Hour">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>160</y>
         <width>43</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>15시</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="time16Hour">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>43</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>16시</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="time17Hour">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>180</y>
         <width>43</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>17시</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="time18Hour">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>43</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>18시</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="time19Hour">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>200</y>
         <width>43</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>19시</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="time20Hour">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>43</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>20시</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="time21Hour">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>220</y>
         <width>43</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>21시</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="time22Hour">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>43</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>22시</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="time23Hour">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>240</y>
         <width>43</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>23시</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QLineEdit" name="keyword_line_edit">
       <property name="geometry">
        <rect>
         <x>2</x>
         <y>0</y>
         <width>231</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Pintrest 검색어</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="3" column="1" colspan="3">
     <widget class="QListWidget" name="listWidget">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="scroll_line_edit">
      <property name="placeholderText">
       <string>스크롤 횟수</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
